src/import-data/ImportData.ts

```diff
@@ -65,16 +65,29 @@ export const ImportData = (props: ImportDataProps): ReactNode => {
   const importPFB = async (importRequest: PFBImportRequest, workspace: WorkspaceInfo) => {
     if (workspace.cloudPlatform === 'Azure') {
       const wdsUrl = await Ajax().Apps.listAppsV2(workspace.workspaceId).then(resolveWdsUrl);
-      await Ajax().WorkspaceData.startImportJob(wdsUrl, workspace.workspaceId, {
+      const { namespace, name, cloudPlatform } = workspace;
```

Comment:
It looks like maybe you already thought of this based on your TODO later on, but it'd be so nice if we could pull the workspace destructuring from here (and later on in the GCP else block) to precede the conditional and apply for both platforms.

Is `cloudPlatform` _not_ set for GCP projects for some reason?  Seems like it'd be helpful if we could assume that it was there.

Reply:
Yeah there's a lot of heartache and headache going on around typing/destructing/etc that I'm hoping will shake out if I can figure out how to typescriptify ImportStatus.js.

Reply:
`cloudPlatform` used to not be set for new workspaces, but I fixed that in #4456.